pilopt/src/lib.rs

```diff
@@ -31,6 +34,48 @@ pub fn optimize<T: FieldElement>(mut pil_file: Analyzed<T>) -> Analyzed<T> {
     pil_file
 }
 
+/// Just perform some very light optimizations (inlining constants and evaluating
+/// constant expressions).
+pub fn optimize_mildly<T: FieldElement>(mut pil_file: Analyzed<T>) -> Analyzed<T> {
```

Comment:
I think a more descriptive name would be nice, maybe `evaluate_constant_expressions`?